spec.html

```diff
@@ -6285,15 +6285,16 @@ <h1>
           _O_: an Object,
           _P_: a property key,
           _V_: an ECMAScript language value,
-        ): a Completion Record normally containing a Boolean
+        ): a Completion Record normally containing ~unused~
```

Comment:
"normally containing" implies to me that sometimes it will contain something else. is this trying to convey that, or that it's a Completion of [[Type]] normal, containing `~unused~`?

Reply:
@ljharb the second thing; see [comment above](https://github.com/tc39/ecma262/pull/2547#issuecomment-940356606).

Reply:
thanks; the comment sums it up - altho i think i'd rather find another term than just settle for defining it.